libc/isystem/stdnoreturn.h

```diff
@@ -0,0 +1,15 @@
+#ifndef LIBC_ISYSTEM_STDNORETURN_H_
+#define LIBC_ISYSTEM_STDNORETURN_H_
+
+#if !(__ASSEMBLER__ + __LINKER__ + 0)
+#if __STDC_VERSION__ + 0 >= 201112
+COSMOPOLITAN_C_START_
+
+#define noreturn _Noreturn
```

Comment:
libc/integral/c.inc defines a token called `wontreturn` so you could make `noreturn` simply be that. We could also potentially update `wontreturn` to use the C11 definition if it's available. Here's what we've got now:

```c
#ifndef wontreturn
#if !defined(__STRICT_ANSI__) &&      \
    (__has_attribute(__noreturn__) || \
     (__GNUC__ + 0) * 100 + (__GNUC_MINOR__ + 0) >= 208)
#define wontreturn __attribute__((__noreturn__))
#else
#define wontreturn
#endif
#endif
```